apps/api/vite.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from 'vite';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	build: {
+		// generate .vite/manifest.json in outDir
+		manifest: true,
+		lib: {
+			entry: './src/main.mts',
+			name: 'pcql-daco-api',
+		},
+		rollupOptions: {
+			input: './src/main.mts',
+			external: ['express'],
+			output: {
+				globals: {
+					express: 'express',
+				},
+			},
+		},
+	},
+	plugins: [],
+});
```

Comment:
vite config for lib. doesn't make sense to me here

Reply:
Yeah I had a feeling this wasn't necessary, let me remove